### PR TITLE
[CUBRIDQA-40] enhance shell fm to integrate into ctp

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 

create a new request to resolve merge fail based on https://github.com/CUBRID/cubrid-testtools/pull/62

so I will close https://github.com/CUBRID/cubrid-testtools/pull/62 and do merge from this request.

thanks

### CODE REVIEW

File: CTP/shell/src/com/navercorp/cubridqa/shell/dispatch/Dispatch.java [hunk 1]
```diff
@@ -58,6 +58,8 @@
 	private Log all;
 	
 	private boolean isFinished;
+	
+	private String toolHome = "";
```
Comment (fanzaiqiang) [inline CTP/shell/src/com/navercorp/cubridqa/shell/dispatch/Dispatch.java]: It's better to remove toolHome and getToolHome() from Dispatch.java.

**Comment by fanzaiqiang** [comment]
@supler 

Please remove package_fail.sh and run_continue.sh.
And also re-generate every kind of cubridqa-xxx.jar.

**Comment by supler** [comment]
@fanzaiqiang 

rebuild and remove all unneeded files/variables/method.

**Comment by fanzaiqiang** [comment]
@supler 

Please go ahead to merge them. Thanks a lot.

**Pull request merged**